sportsref/nfl/players.py

```diff
@@ -233,6 +242,111 @@ def rushing_and_receiving(self, kind='R'):
         df = sportsref.utils.parse_table(table)
         return df
 
+    @sportsref.decorators.memoize
+    @sportsref.decorators.kind_rpb(include_type=True)
+    def defense_and_fumbles(self, kind='R'):
+        """Gets yearly defense/fumble stats for the player.
+
+        :kind: One of 'R', 'P', or 'B'. Case-insensitive; defaults to 'R'.
+        :returns: Pandas DataFrame with defense/fumble stats.
+        """
+        doc = self.get_doc()
+        table = (doc('#all_defense') if kind == 'R'
```

Comment:
I think it's a bit cleaner (and I just made this change myself to the pre-existing functions, will be included in the next release) to change these table assignments to `doc('table#defense')` for example, instead of `doc('#all_defense')`. That's more explicit, and the new utils.get_html should allow you to access table#defense. I realize utils.parse_table works on the div#all_defense div as well, but since this could change, I think it makes the most sense to use table#defense. It'd be great if you could make this change here and on the other functions you added (basically just adding `table` before the `#` and getting rid of `all_`).